### ISSUE TITLE
spacebar installation error on homebrew 3.0.0

### ISSUE THREAD

**Issue by ilevantis** [cmacrae/homebrew-formulae#3]
When installing spacebar with `brew install cmacrae/formulae/spacebar` I get an error. Looks like setting compiler optimization level is now disabled.

```
Error: An exception occurred within a child process:
  MethodDeprecatedError: Calling ENV.O2 is disabled! There is no replacement.
Please report this issue to the cmacrae/formulae tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/cmacrae/homebrew-formulae/spacebar.rb:15
```

**Comment by k4p7** [comment]
toinehartman has a fix: #4

**Comment by k4p7** [comment]
In the meanwhile youcan remove the ENV.02 and then:
`brew install --build-from-source /path/to/spacebar.rb`

**Comment by cmacrae** [comment]
Thanks @kyle-pollock - I've now merged #4
